src/cloud-api-adaptor/test/e2e/nginx_deployment.go

```diff
@@ -22,7 +22,7 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
-const WAIT_NGINX_DEPLOYMENT_TIMEOUT = time.Second * 900
+const WAIT_NGINX_DEPLOYMENT_TIMEOUT = time.Second * 1500
```

Comment:
It seems like a genuine problem. Here we go again... :(

Reply:
Yeah - it seemed like one, but I wanted to double check :(